benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -286,6 +291,19 @@ int main_(std::vector<std::string> const &args, MPI_Comm const comm)
 
 int main(int argc, char *argv[])
 {
+#ifdef ARBORX_ENABLE_CALIPER
+  cali::ConfigManager caliper_manager;
+  std::string caliper_config = "profile.mpi"
+                               ",profile.kokkos"
```

Comment:
I'm not sure yet what happens if caliber is built without mpi or kokkos. Are these simply ignored?

Reply:
yes, it's fine